src/main/views/terms-and-conditions.njk

```diff
@@ -0,0 +1,172 @@
+
+{% set heading = t('Terms and conditions') %}
+
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <p>{{ t('By using this service you’re agreeing to the following terms of use.') }}</p>
+
+      <p> {{ t('This includes the <a href="{{ uri }}">privacy policy</a>.',
+        { uri: AppPaths.privacyPolicyPage.uri }) | safe }}
+      </p>
+
+      <h2 class="heading-medium">{{ t('Who we are') }}</h2>
+
+      <p>
+        {{ t('This service is managed by Her Majesty’s Courts and Tribunals Service and will be '
+        + 'referred to as we from now on.') }}
+      </p>
+
+      <h2 class="heading-medium">{{ t('Information provided by this service') }}</h2>
+
+      <p>
+        {{ t('This service provides information to support you. HMCTS cannot give legal advice on individual cases.') }}
+      </p>
+
+      <p>
+        {{ t('You should answer the questions in the service based on your circumstances, '
+        + 'and seek legal advice if you need it.') }}
+      </p>
+
+      <h2 class="heading-medium">{{ t('Applicable law') }}</h2>
+
+      <p>{{ t('Your use of this service and any dispute arising from its use will be '
+        + 'governed by and construed in accordance with the laws of England and Wales, '
+        + 'including but not limited to the:') }}</p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('Computer Misuse Act 1990') }}</li>
+        <li>{{ t('Data Protection Act 1998') }}</li>
+        <li>{{ t('Mental Capacity Act 2005') }}</li>
+      </ul>
+      <h2 class="heading-medium">{{ t('Responsible use of this service') }}</h2>
+
+      <p>
+        {{ t('The service is designed for people who need to use it or by others '
+        + 'on their behalf and only with their consent.') }}
+      </p>
+
+      <p>
+        {{ t('There are risks in using a shared computer (for example in an internet café) to access this service. '
+        + 'It’s your responsibility to be aware of these risks and to avoid using any computer which may leave your '
+        + 'personal information accessible to others. You’re responsible if you choose to leave a computer unprotected '
+        + 'while using this service.') }}
+      </p>
+
+      <p>
+        {{ t('You must take your own precautions to ensure that the way you access '
+        + 'this service does not expose you to the risk of:') }}
+      </p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('viruses') }}</li>
+        <li>{{ t('malicious computer code') }}</li>
+        <li>{{ t('other forms of interference which may damage your computer system') }}</li>
+      </ul>
+
+      <p>
+        {{ t('You must not misuse this service by knowingly introducing viruses, trojans, worms, logic bombs '
+        + 'or other material which is malicious or technologically harmful. '
+        + 'You must not attempt to gain unauthorised access to this service, '
```

Comment:
No space between end of sentence and the new one.

Reply:
👍